plugins/services/src/js/containers/tasks/ServiceTasksContainer.js

```diff
@@ -38,19 +40,33 @@ class ServiceTasksContainer extends mixin(StoreMixin) {
       || this.state.mesosStateErrorCount !== 0) {
 
       this.setState({
+        errorMessage: null,
         lastUpdate: Date.now(),
         mesosStateErrorCount: 0
       });
     }
   }
 
-  onStateStoreError() {
+  onStateStoreError(xhr) {
+    const errorMessage = Hooks.applyFilter(
```

Comment:
Let's use the mount package instead of filters to allow plugins to hook into the service error view.